profiles/package.mask

```diff
@@ -33,6 +33,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Guillermo Joandet <gjoandet@gmail.com> (2023-03-06)
+# glib beta, in preparation for Gnome 44
+=dev-libs/glib-2.75.4
+=dev-util/glib-utils-2.75.4
+=dev-util/gdbus-codegen-2.75.4
```

Comment:
Let's do this as a single mask for all GNOME 44 components, so it's easy to unmask them all at once:

````
 # Guillermo Joandet <gjoandet@gmail.com> (2023-03-06)
# Mask in preparation for GNOME 44
>=dev-libs/glib-2.75
>=dev-util/gdbus-codegen-2.75
>=dev-util/glib-utils-2.75
````

and also use `>=` with the general version so we don't need to modify the masks if there are further bumps.

Reply:
profiles/package.mask removed from PR, will do a new PR for all Gnome 44 beta